codestyle/checkstyle/checkstyle.xml

```diff
@@ -165,7 +165,7 @@ page at http://checkstyle.sourceforge.net/config.html -->
 
             <property name="ignorePattern"
                       value="${com.puppycrawl.tools.checkstyle.checks.sizes.LineLength.ignorePattern}"
-                      default="^(package .*;\s*)|(import .*;\s*)|( *\* *https?://.*)$"/>
+                      default="^(package .*;\s*)|(import .*;\s*)|( *\* *https?://.*)|( *.*https?://.*)$"/>
```

Comment:
What's the purpose of this change? And you might need to doc this in commit message.

Reply:
Want to ignore comments starting with `*` and contains `http(s)://` link, in order to show in one line.